composer.json

```diff
@@ -15,7 +15,7 @@
 		}
 	],
 	"require": {
-		"php": "7.1.*",
+		"php": ">=7.1",
```

Comment:
Toto není podle mě správné řešení. Mezi desetinnými verzemi dochází ke zpětně nekompatibilním úpravám, viz třeba http://php.net/manual/en/migration72.incompatible.php Některé jsou takové okrajové, ale třeba klíčové slovo `object` bylo celkem zásadní. Podle mě pokud chcete testovat s PHP 7.2 nebo 7.3, je lepší nastavit `7.1.*|7.2.*`. Kdyby teď vyšla nová verze PHP, tato verze se bude tvářit funkčně, ačkoliv není. Pokud už by měla být nějaká výrazná podpora, tak raději `7.*`, která by měla pokrýt opravdu zásadní věci.